simtools/layout/layout_array.py

```diff
@@ -894,19 +895,20 @@ def _get_crs_utm(self):
         """
         if self._epsg:
             crs_utm = pyproj.CRS.from_user_input(self._epsg)
+            self._logger.debug(f"UTM coordinate system: {crs_utm}")
```

Comment:
Are these printed for each telescope? Remember my complaint a few months ago about too many debug messages from the coordinates.

Reply:
No, this is set during the initialization state of the coordinate system (the issue before was mostly about debug messages in `simtools.layout.telescope_position`).

Reply:
OK!